### DISCUSSION TITLE
Support the Development?

### DISCUSSION THREAD

**Discussion by tome9111991** [supermerill/SuperSlicer#1180] [General]
Hi
Is there i way to support the Development?
How i can help the project.
Is there a paypal or patreon link to buy a coffee? :)

**Comment by supermerill** [comment]
yes, there is a github sponsor option (see the sponsor button near the heart)
You can also help with translation and documentation if you have a decent amount of free time.

**Reply by cryd-s** [reply]
where can i post my translation?

**Reply by supermerill** [reply]
You can submit the new po file as a pull request (clone the project, erase the bad .po by your new .po, then commit & push the changes via git and push the "pull request" button on github).
If you don't want to use git, open an issue with "XXXX translation" and post your file in it I'll merge it myself. Note that there is already an Italian and german one.

**Comment by tome9111991** [comment]
Just an idea.
To add a patreon link and/or paypal for single  donation?
I dont like via github

**Reply by supermerill** [reply]
I don't like the paypal idea, as it's too disconnected, and so susceptible to frauds via wrong addresses.

Github has no fee, and patreon has huge fees for low amounts. maybe there is something better than patreon?

**Reply by tome9111991** [reply]
Maybe
https://www.gofundme.com
For one time donate.

**Reply by supermerill** [reply]
gofundme is more like a kickstater-kind of things, it's gated in time.

**Comment by gpread** [comment]
How about [Ko-fi](https://ko-fi.com/) ?

**Reply by supermerill** [reply]
Seems good, i'll try.

**Reply by supermerill** [reply]
Did it works?
https://ko-fi.com/superslicer

**Reply by gpread** [reply]
You tell me, I bought you a coffee, so you should see it now! 😁

**Reply by gpread** [reply]
It was easy to use, just requires

- Email address (to be able to send donator confirm email)
- Credit Card Info (does not appear to accept other payment types currently)
- Optionally you can add a name and/or a message

**Reply by gpread** [reply]
Of course they prefer you to have profile/account and there are benefits. So they send you invite to join once they have your email for the confirm.
But you are not required to, as I did initially (I had been thinking to sign up before and now I did) you can donate anonymously as a guest

**Reply by supermerill** [reply]
> so you should see it now

I receive it! thanks. They even cheer me for receiving my first coffee :-D

I'm going to add it at the sponsor options.

**Comment by PargeLenis** [comment]
Hi @supermerill! I'd love to help with some translation work, but I'm a bit unsure how to translate the word 'you' into German, since the language distinguishes between formal and informal pronouns. Is there a standard or preference?
